### ISSUE TITLE
Home view Recent lists should be semantic lists instead of divs.

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2715]
![image](https://github.com/user-attachments/assets/78832b04-924f-47b2-a575-a6c955098ed0)
